index.html

```diff
@@ -53,6 +54,241 @@ <h1>I'm Redwan.<br/>
+                          </div>
+                          <p>
+                            A daily selection of privately <br/>
+                            personalized reads; no accounts or <br/>
+                            sign-ups required.
+                          </p>     
+                        </div>
+                        <div class="tags">
+                            <ul>
+                                <li>
+                                    <a class="btn" href="">HTML</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">CSS</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">JavaScript</a>
+                                </li>
+                            </ul>
+                          </div>
+                          <div class="action">
+                            <button class="submit">See project</button>
+                          </div>
+                    </div>
+                    <div class="flex-col">
+                        <img
+                        class="snapshoot-portfolio-icon"
+                        alt=""
+                        src="Images/Snapshoot Portfolio (2).png"
+                      />
+                        <div class="top">
+                          <h2>Multi-Post Stories</h2>
+                          <div class="client-parent">
+                            <ul>
+                                <li>CANOPY</li>
+                                <li>Back End Dev </li>
+                                <li>2015</li>
+                            </ul>
+                          </div>
+                          <p>
+                            A daily selection of privately <br/>
+                            personalized reads; no accounts or <br/>
+                            sign-ups required.
+                          </p>     
+                        </div>
+                        <div class="tags">
+                            <ul>
+                                <li>
+                                    <a class="btn" href="">HTML</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">CSS</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">JavaScript</a>
+                                </li>
+                            </ul>
+                          </div>
+                          <div class="action">
+                            <button class="submit">See project</button>
+                        </div>
+                    </div>
+                </div>
+               </div>
+        </section>
+
+
+        <section class="section-bottom"> 
+            <div class="headerTop">
+                <h1>About Myself </h1>
```

Comment:
- [ ] Please change this heading to another tag, you can use **h2---h6**. It is best practice to have a single **h1** in a page because it promotes accessibility, and SEO optimization 👁️‍🗨️